enterprise/migrations/0039_auto_20180122_0934.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-01-22 15:34
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from enterprise.utils import decrypt_string
+
+
+def migrate_passwords(apps, schema_editor):
+    """
+    Migrate password fields to new django-fernet-fields encrypted fields.
+    """
+    EnterpriseCustomerReportingConfiguration = apps.get_model(
+        'enterprise',
+        'EnterpriseCustomerReportingConfiguration'
+    )
+    for config in EnterpriseCustomerReportingConfiguration.objects.all():
+        if config.password:
+            config.decrypted_password = decrypt_string(
+                config.password,
+                config.initialization_vector
+            )
+        if config.sftp_password:
+            config.decrypted_sftp_password = decrypt_string(
```

Comment:
can we remove the utils `encrypt_string` / `decrypt_string` once this is merged and the migration happens?

Reply:
Yes, that is my plan, along with the old password fields.